app/components/Sidebar/components/SidebarLink.tsx

```diff
@@ -158,6 +158,10 @@ const Link = styled(NavLink)<{ $isActiveDrop?: boolean; $isDraft?: boolean }>`
     transition: fill 50ms;
   }
 
+  &:hover svg {
+    display: inline-block;
```

Comment:
very minor comment: does "display: inline" work? because I believe that's the default here